polkadot-parachains/integritee-polkadot/src/xcm_config.rs

```diff
@@ -359,18 +358,18 @@ type Reserves = (
 	ReserveAssetsFrom<AssetHubLocation>,
 );
 
+/// Means for transacting assets on this chain.
+pub type AssetTransactors =
+	(LocalNativeTransactor, ReservedFungiblesTransactor, LocalFungiblesTransactor);
+
 /// The means for routing XCM messages which are not for local execution into the right message
 /// queues.
-pub type XcmRouter = (
+pub type XcmRouter = WithUniqueTopic<(
```

Comment:
Without this, the delivery fee calculation was slightly off. I cannot say why, but some integration tests failed, and then I copied the router from the people's chain.